sys/shell/commands/sc_nanocoap_vfs.c

```diff
@@ -27,6 +27,14 @@
 #include "net/nanocoap_vfs.h"
 #include "shell.h"
 #include "vfs_default.h"
+#include "vfs_util.h"
+
+/**
+ * @brief   Default download location for ncget
```

Comment:
```suggestion
 * @brief   Default download location for ncget
 *
 * @warning Do not forget to add a trailing `/` character 
```

If I understand the `snprintf(... "%s%s", ...)` correctly, this might be a good hint.

Reply:
No, this doesn't need the slash: the %s%s second argument is taken from strrchr output and thus has a slash built in.